src/rules/css/detectBadMozBinding.js

```diff
@@ -2,16 +2,17 @@ import * as messages from 'messages';
 import { LOCAL_CSS_URL } from 'regex';
 
 
-export function detectBadMozBindingURL(rule) {
+export function detectBadMozBindingURL(cssRule, filename= null,
+                                       {startLine, startColumn}={}) {
   var messageList = [];
-  for (let declaration of rule.declarations) {
+  for (let declaration of cssRule.declarations) {
```

Comment:
I think it make sense to clean this up now whilst we're in all these files. 

@Nolski - sorry for adding this to your plate but I think now's a good time since you're touching all the bits that use these vars.